scripts/snapshot-framework-stats.py

```diff
@@ -1,12 +1,13 @@
 #!/usr/bin/env python3
 """Usage:
-    snapshot_framework_stats.py <framework_slug> <stage>
+    snapshot_framework_stats.py <stage> <framework_slug> <filename>
 
 Example:
-    ./snapshot_framework_stats.py g-cloud-7 dev
+    ./snapshot_framework_stats.py production g-cloud-12 myfile.json
```

Comment:
I'd rather avoid `production` in an example, just because this script writes to the DB.

Actually, this is probably a good time to write a 1-line description of what this script does 😸 How about:

`Fetches application stats for a given framework (suppliers, services and users) from the API. The response is stored in an audit event and the output is saved to a JSON file.`